### PR TITLE
Add Example 

### PR DESCRIPTION
**Pull request by komish**
Been playing with the operator and it helped to jot down a top-down view of what preparation might need to take place before a `ResourceLocker` can properly manage a Resource.

Down the line, I imagine this will expand into adding patches and such, but figured I'd get it started. 

Happy to move it to a better place (README.md or some other documentation location) if you prefer.

EDIT: hit enter too soon.

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Usage Examples
+
+We assume that you have already installed the resource-locker-operator in your cluster and the CustomResourceDefinitions are available.
+
+## Locking a resource
+
+In this example, we assume your operator is installed in the recommended namespace:  `resource-locker-operato`. We will create a `ConfigMap` in a second namespace, `resource-locker-test-ns`, locked by a `ResourceLocker` CustomResource.
```
Comment (raffaelespazzoli) [inline EXAMPLES.md:7]: resource-locker-operato -> resource-locker-operator

**Comment by raffaelespazzoli** [comment]
please link page this from the main readme file.

File: EXAMPLES.md (hunk 1 above)
Comment (komish) [reply EXAMPLES.md]: Fixed!

**Comment by komish** [comment]
@raffaelespazzoli I've added a line in the README to point to EXAMPLES. Let me know what else I can do to help!

**Comment by raffaelespazzoli** [comment]
@komish the build is failing, let me try to understand why.

**Comment by raffaelespazzoli** [comment]
@komish please rebase and retry.

**Comment by komish** [comment]
@raffaelespazzoli Rebase completed! ✅

**Comment by komish** [comment]
I've inadvertently deleted the fork associated with this PR but if there are further changes I can look into what's needed to make that happen. I have the original changes pushed to my new fork as a reference.

**Pull request merged**